Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -1268,11 +1270,23 @@ unique_ptr<RWMol> MolDraw2D::setupDrawMolecule(
     rwmol.reset(new RWMol(mol));
     MolDraw2DUtils::prepareMolForDrawing(*rwmol);
   }
+  if (drawOptions().centreMoleculesBeforeDrawing) {
+    if (!rwmol) rwmol.reset(new RWMol(mol));
+    if (rwmol->getNumConformers()) {
+      auto &conf = rwmol->getConformer(confId);
+      RDGeom::Transform3D tf;
+      auto centroid = MolTransforms::computeCentroid(conf);
+      centroid *= -1;
+      tf.SetTranslation(centroid);
+      MolTransforms::transformConformer(conf, tf);
+    }
```

Comment:
I think this is fine, but since we fail below if getNumConformers is empty, why not just to this check earlier.  Also, we will throw an exception of confId is not found, should we handle this here or throw the exception?

Reply:
We could move the test to make sure the molecule has at least one conformer up from 1285, but then the logic in that becomes more complicated (need to do the rwmol vs mol test). Doesn't seem like a net benefit

And, yeah, if the user calls this with a bad conformer id, I think we should throw